tasks/cat2.yml

```diff
@@ -2574,6 +2574,21 @@
       - file_perms
       - rsyslog
 
+# As described in https://access.redhat.com/solutions/66805, the
+# permissions for /var/log/boot.log have to be changed each time the
+# system is rebooted.
+- name: "MEDIUM | V-38623 | PATCH | All rsyslog-generated log files must have mode 0600 or less permissive."
+  lineinfile:
+      dest: /etc/rc.d/rc.local
+      line: chmod 600 /var/log/boot.log
```

Comment:
can you change `chmod 600` to `chmod u-x,go-rwx`?  This is just in case that there is a more restrictive command already in place.